test/modules/types/message_test.js

```diff
@@ -308,4 +309,81 @@ describe('Message', () => {
       assert.deepEqual(actual, expected);
     });
   });
+
+  describe('_mapQuotedAttachments', () => {
+    it('handles message with no quote', async () => {
+      const upgradeAttachment = sinon.stub().throws(new Error("Shouldn't be called"));
+      const upgradeVersion = Message._mapQuotedAttachments(upgradeAttachment);
+
+      const message = {
+        body: 'hey there!',
+      };
+      const result = await upgradeVersion(message);
+      assert.deepEqual(result, message);
+    });
+
+    it('handles quote with no attachments', async () => {
+      const upgradeAttachment = sinon.stub().throws(new Error("Shouldn't be called"));
```

Comment:
Minor: I’ve been using `sinon` sandboxes to ensure stubs, etc. are cleaned up between runs. It’s not a big deal when not dealing with globals, etc. but it might be a nice pattern to adopt for consistency in case other parts of the test will require sandboxes in the future: https://github.com/signalapp/Signal-Desktop/blob/3a05201501dedddcc72b1caf11bc0a26e28a76bc/test/modules/startup_test.js#L8

Reply:
Yep, this would be good to start using when we mock things, especially if they're global. If I'm just generating a stub, I don't see a reason to use it.